src/operator-docs/transformation/mergeMap.ts

```diff
@@ -1,6 +1,92 @@
 import { OperatorDoc } from '../operator.model';
 
 export const mergeMap: OperatorDoc = {
-  'name': 'mergeMap',
-  'operatorType': 'transformation'
+  name: 'mergeMap',
+  operatorType: 'transformation',
+  signature: `public mergeMap(project: function(value: T, ?index: number): ObservableInput,
```

Comment:
Think `index` may be messed up here `?index: number`.

Reply:
@btroncone I didn't get it.

Reply:
I just meant the signature looks off `mergeMap(project: function(value: T, ?index: number)`, should the `?` be there?